mmv1/third_party/terraform/services/workstations/resource_workstations_workstation_config_test.go.erb

```diff
@@ -942,6 +1021,7 @@ resource "google_workstations_workstation_config" "default" {
 
       shielded_instance_config {}
       confidential_instance_config {}
+      boost_configs {}
```

Comment:
If `boost_configs` is empty, this line can be deleted. Otherwise, it should have required fields.

Reply:
fixed, thanks